README.md

```diff
@@ -11,43 +11,42 @@ a Kubernetes cluster at that point of time when a new pod appears for scheduling
 As Kubernetes clusters are very dynamic and their state changes over time, there may be desire
 to move already running pods to some other nodes for various reasons:
 
-* Some nodes are under or over utilized.
-* The original scheduling decision does not hold true any more, as taints or labels are added to
-or removed from nodes, pod/node affinity requirements are not satisfied any more.
-* Some nodes failed and their pods moved to other nodes.
-* New nodes are added to clusters.
+- Some nodes are under or over utilized.
```

Comment:
There's a lot of odd updates to the README.md in this PR, are these intentional or maybe an editor auto-formatting? Would like to undo these if possible